incubator/condor-submit/condor-submit/templates/configmap.yaml

```
@@ -25,12 +25,20 @@ data:
     #!/bin/sh
     # This file will be sourced before the CI Connect provisioner starts up. You
     # can add anything you like here.
-
     # Append PATH
     # Apparently '[ -z "$PATH" ]' doesn't work because of some subtlety where Bash 
     # kind-of-sort-of says $PATH is set, even though it doesn't appear in `env`
     export PATH="$PATH:/bin:/usr/bin:/usr/local/bin:/sbin:/usr/sbin:/usr/local/sbin"
-
     export INNER_GROUP="{{ .Values.UserConfig.ConnectInnerGroup }}"
     export OUTER_GROUP="{{ .Values.UserConfig.ConnectOuterGroup }}"
 
```

```suggestion
---
```
Need this to tell Kubernetes it is looking at a new file now. Each ConfigMap is its own "file" even though we combine them.